daily-tags.sh

```diff
@@ -3,6 +3,22 @@ set -ex
 
 # Check for required variables
 ALIDIST_SLUG=${ALIDIST_SLUG:-alisw/alidist@master}
+if echo "$ALIDIST_SLUG" | grep -q '!!FLPSUITE_LATEST!!'; then
```

Comment:
I would just use FLPSUITE_LATEST. Why the extra `!!`?

Reply:
To make it obvious that it'll be replaced and isn't just a normal tag.

Reply:
Well, it would have been good enough. Anyways, fine. Merging this. Can you update the jenkins job accordingly?

Reply:
(I mean DailyO2Dataflow)